<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="sideBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>151</width>
      <height>641</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="btnRecepcion">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>92</width>
       <height>34</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(145, 111, 255);
</string>
     </property>
     <property name="text">
      <string>Recepcion</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAlmacen">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>92</width>
       <height>34</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(145, 111, 255);
</string>
     </property>
     <property name="text">
      <string>Almacen</string>
     </property>
    </widget>
    <widget class="QWidget" name="subMenuRecepcion" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>120</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(47, 255, 15);</string>
     </property>
     <widget class="QPushButton" name="reservacion">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>92</width>
        <height>34</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(150, 75, 255);
</string>
      </property>
      <property name="text">
       <string>reservacion</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btnAdministracion">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>111</width>
       <height>34</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(145, 111, 255);
</string>
     </property>
     <property name="text">
      <string>Administrador</string>
     </property>
    </widget>
    <widget class="QWidget" name="subMenuAlmacen" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>120</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(47, 255, 15);</string>
     </property>
     <widget class="QPushButton" name="mobiliario">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>92</width>
        <height>34</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(150, 75, 255);
</string>
      </property>
      <property name="text">
       <string>mobiliario</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="subMenuAdministracion" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>120</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(47, 255, 15);</string>
     </property>
     <widget class="QPushButton" name="servicios">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>92</width>
        <height>34</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(150, 75, 255);
</string>
      </property>
      <property name="text">
       <string>Servicios</string>
      </property>
     </widget>
     <widget class="QPushButton" name="equipamiento">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>101</width>
        <height>34</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(150, 75, 255);
</string>
      </property>
      <property name="text">
       <string>Equipamiento</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>811</width>
      <height>641</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QWidget" name="adminServicios">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(165, 255, 248);</string>
     </property>
    </widget>
    <widget class="QWidget" name="adminEquipamiento">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 168, 28);</string>
     </property>
    </widget>
    <widget class="QWidget" name="page_7">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(99, 26, 255);</string>
     </property>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>791</width>
        <height>581</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: #000000;</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>241</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 28px;</string>
        </property>
        <property name="text">
         <string>Tipo de servicios</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>80</y>
          <width>281</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit_2">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>80</y>
          <width>281</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit_3">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>150</y>
          <width>281</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>241</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 28px;</string>
        </property>
        <property name="text">
         <string>Servicios</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
